javacord-core/src/main/java/org/javacord/core/util/handler/channel/ChannelDeleteHandler.java

```diff
@@ -65,12 +63,57 @@ public void handle(JsonNode packet) {
                 logger.debug("Received CHANNEL_DELETE packet for a store channel. These are not supported in this"
                         + " Javacord version and get ignored!");
                 break;
-            default:
-                logger.warn("Unknown or unexpected channel type. Your Javacord version might be out of date!");
+            case GROUP_CHANNEL:
+                logger.info("Received CHANNEL_DELETE packet for a group channel. This should be impossible.");
+                break;
+            case CHANNEL_CATEGORY:
+                handleCategory(packet);
+                break;
+            case PRIVATE_CHANNEL:
+                handlePrivateChannel(packet);
+                break;
+            default: {
+                handleUnknownRegularServerChannel(packet);
+                handleUnknownServerChannel(packet);
```

Comment:
Since the create/update handler have custom error handling as well, for consistency:
```suggestion
                try {
                    handleUnknownRegularServerChannel(packet);
                    handleUnknownServerChannel(packet);
                } catch (Exception exception) {
                    logger.warn("An error occurred when trying to delete a fallback channel implementation", exception);
                }
```

Reply:
It's not only consistency, but like you mentioned in case Discord adds some sort of channel that does not belong to a server  it will throw an exception when trying to get the server id.